back/app/services/xlsx_service.rb

```diff
@@ -173,7 +173,7 @@ def generate_initiatives_xlsx(initiatives, view_private_attributes: false)
       { header: 'author_name',          f: ->(i) { format_author_name i } },
       { header: 'author_email',         f: ->(i) { i.author&.email } },
       { header: 'author_id',            f: ->(i) { i.author_id } },
-      { header: 'cosponsors',           f: ->(i) { i.cosponsors_initiatives.map { |ci| ci.user.email }.join(',') }, skip_sanitization: true },
+      { header: 'cosponsors',           f: ->(i) { i.cosponsors_initiatives.map { |ci| "#{ci.user.email} - status: #{ci.status}" }.join(', ') }, skip_sanitization: true },
```

Comment:
```suggestion
      { header: 'cosponsors',           f: ->(i) { i.cosponsors_initiatives.map { |ci| "#{ci.user.email} - #{ci.status}" }.join("\n") }, skip_sanitization: true },
```

Reply:
Done. Updated screenshot in description.